### ISSUE TITLE
Update the database configuration to use `.env` variables

### ISSUE THREAD

**Issue by paramsiddharth** [ProgramEquity/amplify#432]
# Update the database configuration to use `.env` variables
Currently the database configuration doesn't recognize variables within the `.env` file and only identifies environment variables that are set in the environment. This works for Codespaces because it initializes those variables by default in the workspaces upon startup, but for local development it is a hassle to set those variables everytime.

A potential fix would be updating the [`server/db/util.js`](https://github.com/ProgramEquity/amplify/tree/main/server/db/util.js) file to use `dotenv` to read variables from the `.env` before defining the `getEnv` function.

**Comment by dmilburn** [comment]
we're currently working on making the codespaces experience more seamless and with a higher core count which should hopefully make it so this issue doesn't feel needed so we're going to punt on this one for now.

**Comment by paramsiddharth** [comment]
> we're currently working on making the codespaces experience more seamless and with a higher core count (#464) which should hopefully make it so this issue doesn't feel needed so we're going to punt on this one for now.

I agree. That's understandable. If we manage to make Codespaces more reliable we can put this on temporary hold. It needs to be looked at under #431 i.e. Local development.
